tests/test_food_store_coverage.py

```
@@ -28,6 +28,38 @@ def test_expand_query_none(self):
         result = expand_query(None)
         assert result == []
 
+    def test_search_foods_invalid_limit(self):
+        """Тест search_foods с невалидным limit (строка 44-45)"""
```

Ruff (0.14.1)

Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)